test/charts/progress/progress.spec.js

```diff
@@ -0,0 +1,36 @@
+describe('Directive: pfPercentageUsed', function() {
```

Comment:
Shouldn't the name of this file match the name of the directive?  I would think this should be named `pf-percentage-used.spec.js`.